lxd/instance/drivers/driver_qemu.go

```diff
@@ -2724,6 +2814,70 @@ func (d *qemu) Stop(stateful bool) error {
 		return err
 	}
 
+	// Handle stateful stop.
+	if stateful {
+		os.Remove(d.StatePath())
```

Comment:
@stgraber same about adding a separate function.

Reply:
@stgraber oh I just saw your comment about keeping this inline until snapshots are supported, ignore please.